jet_fuel/test/integration/url_usage_test.rb

```diff
@@ -0,0 +1,43 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.